docs/websites/nginx/install-nginx-and-a-startssl-certificate-on-debian-8-jessie.md

```diff
@@ -0,0 +1,299 @@
+
+        cd /opt
+        wget http://nginx.org/download/nginx-1.10.1.tar.gz
+
+6.  Extract the file, then navigate to the new directory:
+
+        tar -zxvf nginx-1.10.1.tar.gz
+        cd /opt/nginx-1.10.1
+
+7.  Configure the build options. The options shown here provide a good starting point, and will allow HTTP/2 compatibility. However, you may wish to add [other options](https://www.nginx.com/resources/admin-guide/installing-nginx-open-source/) as well, depending on your needs:
+
+        ./configure --prefix=/opt/nginx --conf-path=/etc/nginx/nginx.conf --user=nginx --group=nginx --with-debug --with-threads --with-http_ssl_module --with-ipv6 --with-http_v2_module
+
+    You will see an output of your configuration summary when this step is complete. Be sure to note the given file paths since you'll need them shortly.
+
+8.  Build and install nginx using the options you specified in Step 7:
+
+        make
+        make install
+
+9.  Create an nginx system user:
+
+        sudo adduser --system --no-create-home --disabled-login --disabled-password --group nginx
+
+10. Create a systemd service script to run Nginx:
+
+    {: .file}
+    /lib/systemd/system/nginx.service
+    :   ~~~ shell
+        [Unit]
+        Description=A high performance web server and a reverse proxy server
+        After=network.target
+
+        [Service]
+        Type=forking
+        PIDFile=/opt/nginx/logs/nginx.pid
+        ExecStartPre=/opt/nginx/sbin/nginx -t -q -g 'daemon on; master_process on;'
+        ExecStart=/opt/nginx/sbin/nginx -g 'daemon on; master_process on;'
+        ExecReload=/opt/nginx/sbin/nginx -g 'daemon on; master_process on;' -s reload
+        ExecStop=-/sbin/start-stop-daemon --quiet --stop --retry QUIT/5 --pidfile /opt/nginx/logs/nginx.pid
+        TimeoutStopSec=5
+        KillMode=mixed
+
+        [Install]
+        WantedBy=multi-user.target
+        ~~~
+
+    If you used the build configuration options shown in Step 7, you may use this script exactly as shown. However, if you specified your own file paths, you may need to adjust the value for `PIDFile` to match your PID file, and the paths in the lines beginning with `Exec` to match your nginx binary file path. These files paths were included in the output in Step 7.
+
+11. Make the `nginx.service` script executable:
+
+        chmod +x /lib/systemd/system/nginx.service
+
+12. Start nginx:
+
+        systemctl start nginx
+
+    Optionally, you can enable it to start automatically on boot:
+
+        systemctl enable nginx
+
+{: .note}
+>If you compile from a source distribution as above, some of the files referenced in this and other nginx guides may not be created by default. You may create those files yourself at their specified file paths and nginx will work as intended. For more information, refer to our guide on [how to configure nginx](/docs/websites/nginx/how-to-configure-nginx).
```

Comment:
This isn't enough. I've spent half an hour searching down and creating files as needed based on the errors when starting the service. Now I have:

```
# systemctl status nginx.service
● nginx.service - A high performance web server and a reverse proxy server
   Loaded: loaded (/lib/systemd/system/nginx.service; enabled)
   Active: failed (Result: timeout) since Fri 2016-07-22 18:41:33 EDT; 1min 5s ago
  Process: 6980 ExecStart=/opt/nginx/sbin/nginx -g daemon on; master_process on; (code=exited, status=0/SUCCESS)
  Process: 6979 ExecStartPre=/opt/nginx/sbin/nginx -t -q -g daemon on; master_process on; (code=exited, status=0/SUCCESS)

Jul 22 18:40:03 petrucci systemd[1]: PID file /opt/nginx/logs/nginx.pid not readable (yet?) after start.
Jul 22 18:41:33 petrucci systemd[1]: nginx.service start operation timed out. Terminating.
Jul 22 18:41:33 petrucci systemd[1]: Failed to start A high performance web server and a reverse proxy server.
Jul 22 18:41:33 petrucci systemd[1]: Unit nginx.service entered failed state.
```

Creating the `pid` file manually does not solve the issue. If we're going to give instructions that involve compiling nginx from source, we need to include everything it takes to make it run, especially considering that compiling is required to satisfy the needs of the primary guide in this PR.